src/main.ts

```diff
@@ -2,12 +2,18 @@ import Vue from "vue";
 import App from "./App.vue";
 import router from "./router";
 import vuetify from "./plugins/vuetify";
+import io from "socket.io-client";
+import VueSocketIOExt from "vue-socket.io-extended";
+
+const socket = io("http://socketserver.com:3000");
```

Comment:
This looks like you're establishing a socket connection with the server even sooner than any Vue component is created. I would suggest creating a socket connection after a player creates/joins a room. That would prevent creating socket connections for let's say web crawlers

Reply:
@azirbrooks Good point. I will keep that in mind for the real implementation.